blog/2023-10-11-two-new-core-team-members.md

```diff
@@ -0,0 +1,23 @@
+# Introducing Single-Spa's New Core Team Members
+
+Please welcome the latest additions to the Single-Spa core team. These talented individuals bring fresh energy and a wealth of expertise and are poised to revitalize and expand the Single-Spa project.
+
+## Meet the Team
+
+**[Artur Androsovych](https://github.com/arturovt)**
+
+Artur Androsovych is a Google Developer Expert in Angular and an open-source contributor who has been focusing on runtime performance and teaching teams about Angular internals for the past few years. He has maintained the single-spa-angular project for years, and we're excited for him to join the core team.
+
+**[Roberto Mosca](https://github.com/robmosca)**
+
+Hey there! I’m a Principal Software Engineer at TravelPerk (www.travelperk.com), where I’m the go-to person for all things related to our frontend platform. With a full-stack background, I’ve got my hands on a variety of tech—from Python to NodeJS, React, Webpack... Before diving into the world of business travel, I was immersed in biomedical research as a bioinformatician. When I’m not in front of a screen, you’ll find me swimming, cooking up some Italian delicacies, or tinkering with my Rubik’s cube-solving robots. And when the stress levels rise? Knitting is my zen.
```

Comment:
To harmonize this with the other two descriptions, I propose the following modification:
```suggestion
Roberto is a Principal Software Engineer at TravelPerk (www.travelperk.com), where he is the go-to person for all things related to their frontend platform. With a full-stack background, he's got his hands on a variety of tech — from Python to NodeJS, React, Webpack... Before diving into the world of business travel, he was immersed in biomedical research as a bioinformatician. When he is not in front of a screen, you’ll find him swimming, cooking up some Italian delicacies, or tinkering with his Rubik’s cube-solving robots.
```